### ISSUE TITLE
[QA] [Mainnet] Cannot send asset between Desktop app wallets

### ISSUE THREAD

**Issue by tsadovska** [wormhole-foundation/wormhole-connect#1338]
## Describe the current behavior in detail
Checked on https://wormhole-connect-mainnet.netlify.app/ [Mainnet] with Google Chrome Version 120.0.6099.109 (Official Build) (x86_64).

Steps:
1. Prepare to send 0.0001 AVAX from Avalanche (Zerion) to Celo (Zerion)
2. Click "Approve and proceed with transaction" button

Note: Zerion wallet is connected through WalletConnect Desktop app

Actual result:
Notification to approve transaction is not displayed in Zerion wallet. Transaction fails with error:
```
Error: Missing or invalid. request() method: wallet_addEthereumChain
    at Ha.isValidRequest (main.ff0584f4.js:2:10883970)
    at async Ha.request (main.ff0584f4.js:2:10867433)
    at async Wa.request (main.ff0584f4.js:2:10890439)
    at async Is.request (main.ff0584f4.js:2:10969451)
    at async js.request (main.ff0584f4.js:2:10980978)
    at async tc.request (main.ff0584f4.js:2:10989908)
    at async v4.evm.mO.walletConnect.connectorOptions.projectId.addChain (main.ff0584f4.js:2:18661018)
    at async v4.evm.mO.walletConnect.connectorOptions.projectId.switchChain (main.ff0584f4.js:2:18660784)
    at async w4 (main.ff0584f4.js:37:280414)
    at async Object.onClick (main.ff0584f4.js:137:4924)
```

![image](https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/7278ce18-fd54-4cf2-976a-8391edaed4a5)

https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/8b827dd4-9548-4b77-8419-47d4553be4b4

## What is the expected/desired behavior
Transaction is successful.

**Comment by tsadovska** [comment]
Issue is also reproducible on https://preview.portalbridge.com/7102d8ccad/ [Mainnet] when sending 0.0001 AVAX from Avalanche (Zerion) to Celo (Zerion).

Error:
```
Error: Missing or invalid. request() method: wallet_addEthereumChain
    at Ha.isValidRequest (main.js:2:10880239)
    at async Ha.request (main.js:2:10863702)
    at async Wa.request (main.js:2:10886708)
    at async Is.request (main.js:2:10965720)
    at async js.request (main.js:2:10977247)
    at async tc.request (main.js:2:10986177)
    at async v4.evm.mO.walletConnect.connectorOptions.projectId.addChain (main.js:2:18658782)
    at async v4.evm.mO.walletConnect.connectorOptions.projectId.switchChain (main.js:2:18658548)
    at async w4 (main.js:37:280414)
    at async Object.onClick (main.js:137:4924)
```

![image](https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/3069388d-ae66-4eb2-829d-845f1711ebff)

**Comment by tsadovska** [comment]
Issue is also reproducible with Infinity Desktop app wallet. 

Checked with:
*  Google Chrome Version 120.0.6099.109 (Official Build) (x86_64)
* Infinity wallet version 2.2.0

**Comment by tsadovska** [comment]
Issue is reproducible on https://preview.portalbridge.com/4adb101c39/ [Mainnet] when sending 0.0001 AVAX from Avalanche (Zerion) to Fantom (Zerion). 

Error:
```
Error: Missing or invalid. request() method: wallet_addEthereumChain
    at Ha.isValidRequest (main.js:2:10880239)
    at async Ha.request (main.js:2:10863702)
    at async Wa.request (main.js:2:10886708)
    at async Is.request (main.js:2:10965720)
    at async js.request (main.js:2:10977247)
    at async tc.request (main.js:2:10986177)
    at async P4.evm.xO.walletConnect.connectorOptions.projectId.addChain (main.js:2:18663928)
    at async P4.evm.xO.walletConnect.connectorOptions.projectId.switchChain (main.js:2:18663694)
    at async T4 (main.js:37:280414)
    at async Object.onClick (main.js:137:4924)
```

**Comment by tsadovska** [comment]
Issue will be rechecked after https://github.com/wormhole-foundation/wormhole-connect/issues/1495 is resolved.

**Comment by agodlevska** [comment]
Issue is still blocked by https://github.com/wormhole-foundation/wormhole-connect/issues/1495.

**Comment by tsadovska** [comment]
Closing as duplicate of https://github.com/wormhole-foundation/wormhole-connect/issues/1495.
